lib/util/problem.js

```diff
@@ -117,6 +119,9 @@ const problems = {
     // warning: there are pending submissions
     pendingSubmissions: problem(400.29, ({ count }) => `There are ${count} pending submissions`),
 
+    // { expected: "list of expected parameters", actual: "list of provided parameters" }
+    unexpectedAttributes: problem(400.30, ({ expected, actual }) => `Expected parameters: (${expected.join(', ')}). Got (${actual.join(', ')}).`),
```

Comment:
Problem codes can't be divisible by 10 because 400.30 === 400.3. So we can skip here to 400.31.